android/src/main/java/com/swmansion/gesturehandler/core/NativeViewGestureHandler.kt

```diff
@@ -278,4 +285,14 @@ class NativeViewGestureHandler : GestureHandler<NativeViewGestureHandler>() {
   private class ScrollViewHook : NativeViewGestureHandlerHook {
     override fun shouldCancelRootViewGestureHandlerIfNecessary() = true
   }
+
+  private class ReactViewGroupHook : NativeViewGestureHandlerHook {
+    // In some cases wrapping component with NativeViewGestureHandler doesn't work, because the component
+    // itself has a wrapper. This happens for example with WebView, where it has 2 layers of wrappers above
+    // an actual WebView.
+    //
+    // Calling onTouchEvent won't do anything on wrappers, therefore we use dispatchTouchEvent. This way
+    // not only do we trigger onTouchEvent, but also pass event to children.
```

Comment:
```suggestion
    // There are cases where a native component is wrapped with a `ReactViewGroup` (the component is rendered 
    // inside a `<View />` component in JS). In such cases, calling `onTouchEvent` wouldn't work as those are 
    // ignored by the wrapper view. Instead `dispatchTouchEvent` can be used, which causes the view to dispatch 
    // the event to its children.
```

I'd avoid explicitly mentioning implementation details of other libraries as those can change independently of RNGH and cause the comment to be obsolete.